src/Pages/AccountsdetailsPage/AccountdetailsPage.js

```diff
@@ -0,0 +1,75 @@
+import { html, css, LitElement } from 'lit-element';
+import { FeServices } from '../../FeServices.js';
+
+export class AccountdetailsPage extends LitElement {
+  static get styles() {
+    return css`
+      .cards {
+        display: flex;
+        margin-top: 100px;
+
+        justify-content: center;
+      }
+      .footer {
+        justify-content: center;
+        margin-top: 20px;
+      }
+      .button {
+        margin-left: 100px;
+      }
+      legend {
+        font-size: 18px;
+      }
+    `;
+  }
+
+  static get properties() {
+    return {
+      data: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+    this.data = [];
+  }
+
+  async firstUpdated() {
+    const data = await FeServices.getRequest({
+      url: '/accountinfo/925548553975232',
+    });
+
+    console.log('requested data', data);
+    return data;
```

Comment:
you dont need to return the data, you need to store it in `this.data` bcz it was used in the page